automatic/qtcreatorcdbext/qtcreatorcdbext.nuspec

```
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Do not remove this test for UTF-8: if “Ω” doesn’t appear as greek uppercase omega letter enclosed in quotation marks, you should use an editor that supports UTF-8, not this one. -->
+<package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
+  <metadata>
+    <id>qtcreatorcdbext</id>
+    <version>4.13.2</version>
+    <packageSourceUrl>https://github.com/admiringworm/chocolatey-packages/tree/master/automatic/qtcreatorcdbext</packageSourceUrl>
+    <owners>AdmiringWorm</owners>
+    <title>Qt Creator CDB Extension</title>
+    <authors>Qt Project</authors>
+    <projectUrl>http://wiki.qt.io/Qt_Creator</projectUrl>
+    <iconUrl>https://cdn.jsdelivr.net/gh/AdmiringWorm/chocolatey-packages@35eb3e49edb3411d779f64e57ef023abac8a3a06/icons/qtcreator.png</iconUrl>
+    <copyright>© 2020 The Qt Company Ltd.</copyright>
+    <licenseUrl>http://code.qt.io/cgit/qt-creator/qt-creator.git/plain/LICENSE.GPL3-EXCEPT?h=4.1</licenseUrl>
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
+    <projectSourceUrl>http://code.qt.io/cgit/qt-creator/qt-creator.git/</projectSourceUrl>
+    <docsUrl>https://doc.qt.io/qtcreator/</docsUrl>
```

This url should be

```suggestion
    <docsUrl>https://wiki.qt.io/Qt_Creator_Windows_Debugging</docsUrl>
```

Which is the only location I can find any documentation of the extension.

---

Done.